src/main.cpp

```diff
@@ -3217,15 +3403,57 @@ bool ActivateBestChain(CValidationState &state, const CChainParams& chainparams,
         const CBlockIndex *pindexFork;
         bool fInitialDownload;
         {
-            LOCK(cs_main);
             CBlockIndex *pindexOldTip = chainActive.Tip();
             pindexMostWork = FindMostWorkChain();
 
+            // This is needed for PV because FindMostWorkChain does not necessarily return the block with the lowest nSequenceId
+            if (fParallel)
+            {
```

Comment:
egyptian braces and new line braces are both used in this codebase. I think we should strive for consistency. While semantic is the same in C++, semantic differs in JS and we are using it for the website part, so I'd advocate for egyptian braces.